apis/nucleus/src/components/listbox/ListBox.jsx

```diff
@@ -144,15 +158,19 @@ export default function ListBox({
                 }))
               )
               .then((p) => {
+                let losPaginos = p;
+                if (filterValues && p && p.length) {
+                  losPaginos = applyValueFilter(p, filterValues);
```

Comment:
Is it possible to modify `p` directly instead of creating `losPaginos` ?

Reply:
Possible, but the lint would complain and immutability is usually preferred.